src/components/site/SummaryList.js

```diff
@@ -3,7 +3,7 @@ import React from "react";
 const SummaryList = props => {
   const { items = [] } = props;
   return (
-    <React.Fragment>
+    <div>
       {items.map(item => (
         <div key={item.key} className="dg_summary-list">
```

Comment:
this was right, `dg_summary-list__item`

The reason this is is its something that can be repeated (inside map).

final markup should be

```html
<div class="dg_summary-list"> <!-- I am the container for a list of items, this is denoted by the "list" word-->
  <div class="dg_summary-list__item">stuff goes here</div>
  <div class="dg_summary-list__item">stuff goes here</div>
  <div class="dg_summary-list__item">stuff goes here</div>
  <div class="dg_summary-list__item">stuff goes here</div>
  <div class="dg_summary-list__item">stuff goes here</div>
  <div class="dg_summary-list__item">stuff goes here</div>
</div>
```

this pattern comes from

```html
<ul>
  <li>stuff goes here</li>
  <li>stuff goes here</li>
  <li>stuff goes here</li>
  <li>stuff goes here</li>
</ul>
```

where the container elment is called an `unordered list`, and each `li` is named a `list item`

In react, most things inside of a map function like you have can be called an "item".

Does that make sense?

Reply:
you must have gotten the old code but it does have the actual pattern you describe.